update_vendored_collections/update_vendored_collections.py

```diff
@@ -104,6 +119,44 @@ def spec_add_changelog(content, collection_tarballs, lines):
             return content
 
 
+def configure_fkinit(credentials_file, repo):
+    print("Configure Fedora kerberos")
+    hsh = yaml.safe_load(open(credentials_file))
+    credentials = {}
+    for credential in hsh["credentials"]:
+        credentials.update(credential)
+    cmd = ["fkinit", "--user", credentials["username"]]
+    run_cmd(cmd, repo, credentials["password"])
```

Comment:
Is there no token based access?  It has to be kinit?

Reply:
How does packit do this?  Does it also use kinit?

Reply:
Yes it requires Fedora token, I am adding it to BaseOS CI Openshift. It will then do `sed -i "s/<password>/${FEDORA_PASSWORD}/" fedora_credentials.yml`

Reply:
And this PR adds `update_vendored_collections/fedora_credentials.yml`. Openshift will have the token stored in ${FEDORA_PASSWORD}

Reply:
Packit uses a token when used as a CLI: https://packit.dev/docs/configuration/#values . I don't know about Packit as a service, but I suppose it is using a token as well, storing the Kerberos password somewhere sounds like a bad idea.